packages/components-native/package.json

```diff
@@ -70,7 +70,7 @@
     "date-fns-tz": "^2.0.0",
     "react": "^18.2.0",
     "react-intl": "^6.4.2",
-    "react-native": ">=0.69.2",
+    "react-native": ">=0.77.0",
```

Comment:
doing the opposite of this comment https://github.com/GetJobber/atlantis/pull/2540#issuecomment-2860211146

after talking with Scott, we've decided it's safer and better for Jobber for this to closer match the actual version we are building with

keeping it as >= to avoid blocking the app, but worth noting we can't actually guarantee compatibility with say 0.78 yet

we might work fine with it, we might not. it's unclear right now. 

bumping the peer deps is a breaking change though so I'll update the title and tagging on both of these PRs

Reply:
with this change, it'll be important to try installing the package in product

I'll add another comment and trigger a new prerelease build once we get a passing CI run

Reply:
ah, yeah this is no good. it's too high, our own app is slightly behind this version. I'll reduce it slightly.